tronbyt_server/manager.py

```diff
@@ -397,7 +399,11 @@ def update(device_id: str) -> ResponseReturnValue:
 
             return redirect(url_for("manager.index"))
     device = g.user["devices"][device_id]
-    device["ws_url"] = ws_root() + f"/{device_id}/ws"
+
+    # Ensure ws_url is set for all devices (for existing devices)
```

Comment:
This feels icky to me because it does an `update()` call to retroactively set `ws_url` for existing devices. Not sure if there's a better way?